Mathlib/Analysis/Complex/UpperHalfPlane/Basic.lean

```diff
@@ -120,6 +120,14 @@ lemma I_re : I.re = 0 := rfl
 @[simp, norm_cast]
 lemma coe_I : I = Complex.I := rfl
 
+open Complex in
+lemma pnat_div_upperHalfPlane_im_pos (n : ℕ+) (z : ℍ) : 0 < (-(n : ℂ) / z).im := by
+  simp only [div_im, neg_im, natCast_im, neg_zero, coe_re, zero_mul, zero_div, neg_re, natCast_re,
+    coe_im, neg_mul, zero_sub, Left.neg_pos_iff, div_neg_iff, Left.neg_neg_iff, Nat.cast_pos,
+    PNat.pos, mul_pos_iff_of_pos_left, Complex.normSq_pos, ne_eq]
+  right
+  simpa using ⟨z.2, ne_zero z⟩
```

Comment:
This is in the `UpperHalfPlane` namespace so you don't really need `UpperHalfPlane` in the statment. My instinct would be to formulate using `Nat` with a `[NeZero]` typeclass assumption, rather than `PNat`, but YMMV. 

If you move this to a few lines later in the file (maybe after `ne_int`) then you can do the following:
```suggestion
lemma im_pnat_div_pos (n : ℕ) [NeZero n] (z : ℍ) : 0 < (-(n : ℂ) / z).im := by
  suffices 0 < n * z.im / Complex.normSq z by simpa [Complex.div_im, neg_div]
  positivity [NeZero.ne n, z.normSq_pos]
```

Reply:
No, I just first did it as a batch and then was doing some of the other ones by hand, including moving this one. It should be fine now.